enumerable_methods.rb

```diff
@@ -106,14 +115,16 @@ def my_none?
   end
 
   def my_count
-    return to_enum unless block_given?
-
     arr = self
     if arr.is_a? Array
       x = 0
       i = 0
       while i < arr.length
-        x += 1 if yield(arr[i])
+        if block_given?
+          x += 1 if yield(arr[i])
+        else
+          x += 1 if arr[i]
+        end
         i += 1
       end
       x
```

Comment:
**[REQUIRED]**
* Counts the number of items in enum that are equal to item if an argument is given.
```
Failure/Error: expect(array.my_count(LOWEST_VALUE)).to eq array.count(LOWEST_VALUE)
      
      ArgumentError:
        wrong number of arguments (given 1, expected 0)
```